site/guides/porch-installation.md

```diff
@@ -190,7 +190,7 @@ As above, you can verify that Porch is running by querying the `api-resources`:
 kubectl api-resources | grep porch
 ```
 
-And start [using Porch](./porch-user-guide.md) if the Porch resources are
+And start [using Porch](guides/porch-user-guide.md) if the Porch resources are
```

Comment:
In my experience, some of the links within the site didn't work with the `.md` extension, and many links I encountered were absolute. that said, LGTM if it works though I'd assume `/guides/porch-user-guide` is what we need.

Reply:
I have verified these changes running the site locally. I will verify that this works on the deployed site as well once it is merged.